source/standards/source-code.html.md.erb

```diff
@@ -11,25 +11,29 @@ At GDS, we follow the principles set out in the Service Manual for managing the
 - [using version control](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding)
 - [making source code open](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable)
 
-You must [keep some data and code closed](https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed), including keys and credentials, algorithms used to detect fraud, and code or data that makes clear details of unannounced policy.
+You must [keep some data and code closed](https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed), including:
+
+- keys and credentials
+- algorithms used to detect fraud
+- code or data that makes clear details of unannounced policy
 
 ## Use GitHub
 
 [GDS uses GitHub as its code repository](https://gdstechnology.blog.gov.uk/2016/05/31/how-we-use-git-at-the-government-digital-service/).
 
-Place new repositories for GDS services in the [alphagov](https://github.com/alphagov/) organisation on GitHub.
+Put new repositories for GDS services in the [alphagov](https://github.com/alphagov/) organisation on GitHub.
 
-You can use your personal GitHub account to access alphagov. Ask your tech lead or technical architect to invite you.
+You can use your personal GitHub account to access alphagov. Ask your tech lead or technical architect to invite you. GDS will revoke your access to alphagov when you leave GDS.
 
 To secure your Github repository, make sure you:
 
 - configure two-factor authentication for your account
-- ensure Git repositories are backed up to another location (this should be a team responsibility)
+- backup Git repositories to another location (this should be a team responsibility)
```

Comment:
I don't know any teams that do this. Each repo is on every developer's machine, so it's mostly unnecessary?

Reply:
So is it something we should be doing? Or is the repo being on local machines enough?

Reply:
GOV.UK do this with a gitlab instance, I believe

Reply:
Ok, I'm going to change this to an optional thing to do rather than a must. Thanks both.